_rules/visible-label-in-accessible-name-2ee8b8.md

```diff
@@ -65,39 +75,39 @@ The understanding document of [2.5.3 Label in Name][understand253] use the term
 
 #### Passed Example 1
 
-This link has [visible][] text that matches the [accessible name][].
+This link has [visible inner text][] that is equal to the [accessible name][].
 
 ```html
 <a href="https://act-rules.github.io/" aria-label="ACT rules">ACT rules</a>
 ```
 
 #### Passed Example 2
 
-This link has [visible][] text that, ignoring trailing whitespace, matches the [accessible name][].
+This link has [visible inner text][] that, ignoring whitespace, is equal to the [accessible name][].
 
 ```html
-<a href="https://act-rules.github.io/" aria-label="  ACT rules  ">ACT rules</a>
+<a href="https://act-rules.github.io/" aria-label="  ACT   rules  ">ACT rules</a>
 ```
 
 #### Passed Example 3
 
-This link has [visible][] text that, ignoring case, matches the [accessible name][].
+This link has [visible inner text][] that, ignoring case, is equal to the [accessible name][].
 
 ```html
-<a href="https://act-rules.github.io/" aria-label="act rules">ACT rules</a>
+<a href="https://act-rules.github.io/" aria-label="act Rules">ACT rules</a>
 ```
 
 #### Passed Example 4
 
-This button has [visible][] text that is contained within the [accessible name][].
+This button has [visible inner text][] that is contained within the [accessible name][] according to the [label in name algorithm][].
 
 ```html
 <button aria-label="Next Page in the list">Next Page</button>
 ```
 
 #### Passed Example 5
 
-This button has [visible][] text that does not need to be contained within the [accessible name][], because the "x" text node is [non-text content][]. Note: this would need to meet SC 1.1.1 Non text content.
+This button has [visible inner text][] that does not need to be contained within the [accessible name][], because the "x" text node is [non-text content][]. Note: this would need to meet SC 1.1.1 Non text content.
```

Comment:
In the algorithm, 

> For each character that either a) represents non-text content, or b) isn't a letter or a digit: replace that character with a space character.

If "X" is replaced with a space char, why would this example pass?